diff -up libcgroup-0.36.1/scripts/init.d/cgconfig.in.lsb1 libcgroup-0.36.1/scripts/init.d/cgconfig.in
--- libcgroup-0.36.1/scripts/init.d/cgconfig.in.lsb1	2010-07-14 13:19:52.000000000 +0200
+++ libcgroup-0.36.1/scripts/init.d/cgconfig.in	2010-07-14 13:19:52.000000000 +0200
@@ -121,6 +121,12 @@ start() {
 
         if [ $? -eq 0 ]
         then
+                if [ ! -s $CONFIG_FILE ]
+                then
+                    log_failure_msg $CONFIG_FILE "is not configured"
+                    return 6
+                fi
+
                 $CGCONFIGPARSER_BIN -l $CONFIG_FILE
                 retval=$?
                 if [ $retval -ne 0 ]
@@ -202,8 +208,10 @@ case $1 in
     'status')
         if [ -f /var/lock/subsys/$servicename ] ; then
             echo "Running"
+            exit 0
         else
             echo "Stopped"
+            exit 3
         fi
 	;;
     *)
diff -up libcgroup-0.36.1/scripts/init.d/cgred.in.lsb1 libcgroup-0.36.1/scripts/init.d/cgred.in
--- libcgroup-0.36.1/scripts/init.d/cgred.in.lsb1	2010-07-14 13:19:52.000000000 +0200
+++ libcgroup-0.36.1/scripts/init.d/cgred.in	2010-07-14 13:32:36.000000000 +0200
@@ -70,14 +70,21 @@ RETVAL=0
 
 start()
 {
-	echo $"Starting CGroup Rules Engine Daemon..."
+	echo -n $"Starting CGroup Rules Engine Daemon: "
 	if [ -f "/var/lock/subsys/$servicename" ] ; then
 		log_failure_msg "$servicename is already running with PID `cat ${pidfile}`"
-		return 1
+		return 0
+	fi
+	if [ ! -s /etc/cgrules.conf ]; then
+		log_failure_msg "not configured"
+		return 6
 	fi
 	daemon --check $servicename --pidfile $pidfile $CGRED_BIN $OPTIONS
 	RETVAL=$?
 	echo
+	if [ $RETVAL -ne 0 ]; then
+		return 7
+	fi
 	[ $RETVAL -eq 0 ] && touch /var/lock/subsys/$servicename
 	echo "`pidof $processname`" > $pidfile
 }
@@ -85,6 +92,10 @@ start()
 stop()
 {
 	echo -n $"Stopping CGroup Rules Engine Daemon..."
+	if [ ! -f $pidfile ]; then
+		log_success_msg
+		return 0
+	fi
 	killproc -p $pidfile $processname -TERM
 	RETVAL=$?
 	echo
@@ -92,19 +103,20 @@ stop()
 		rm -f /var/lock/subsys/$servicename
 		rm -f $pidfile
 	fi
-	log_success_msg
 }
 
 # See how we are called
 case "$1" in
 	start)
 		start
+		RETVAL=$?
 		;;
 	stop)
 		stop
+		RETVAL=$?
 		;;
 	status)
-		status -p $pidfile $processname
+		status -p $pidfile $servicename
 		RETVAL=$?
 		;;
 	restart)
